microservice/src/test/java/com/skytouch/MicroserviceApplicationTests.java

```diff
@@ -0,0 +1,61 @@
+package com.skytouch;
+
+
+import com.skytouch.common.model.Product;
+import com.skytouch.model.ProductEntity;
+import com.skytouch.repository.ProductRepository;
+import com.skytouch.service.ProductService;
+import org.dozer.DozerBeanMapper;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.util.List;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class MicroserviceApplicationTests {
```

Comment:
Is it needed for the class to be public?

Reply:
Yes, the file should have one public class.

Reply:
Could you please elaborate? 
Give us an explanation why class needs to be public

Reply:
it's necessary because the class publicClassValidator in JunitFramework, validate that the unitTest class is public.
https://junit.org/junit4/javadoc/4.12/org/junit/validator/PublicClassValidator.html